docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -282,36 +396,73 @@ When you define an aggregation in a {dfeed}, it must have the following form:
 ----------------------------------
 // NOTCONSOLE
 
+When using a `composite` aggregation:
+
+[source,js]
+----------------------------------
+"aggregations": {
+  "composite_agg": {
+    "sources": [
+      {
+        "date_histogram_agg": {
+          "field": "time",
+          ...settings...
+        }
+      },
+      ...other valid sources...
+      ],
+      ...composite agg settings...,
+      "aggregations": {
+        "timestamp": {
+            "max": {
+              "field": "time"
+            }
+          },
+          ...other aggregations...
+          [
+            [,"aggregations" : {
+              [<sub_aggregation>]+
+            } ]
+          }]
+      }
+   }
+}
+----------------------------------
+// NOTCONSOLE
+
 The top level aggregation must be either a
 {ref}/search-aggregations-bucket.html[bucket aggregation] containing as single
-sub-aggregation that is a `date_histogram` or the top level aggregation is the
-required `date_histogram`. There must be exactly one `date_histogram` 
-aggregation. For more information, see
-{ref}/search-aggregations-bucket-datehistogram-aggregation.html[Date histogram aggregation].
+sub-aggregation that is a `date_histogram`, the top level aggregation is the
```

Comment:
```suggestion
sub-aggregation that is a `date_histogram`
```

Reply:
I may be missing something, but removing `, the top level aggregation is the` misses the point. 

The requirements are:

- A `date_histogram` sub aggregation
- A top level `date_histogram` aggregation
- A top level `composite` aggregation

These are all exclusive requirements. No overlap is allowed and exactly one should be satisfied.

Reply:
@lcawl I think i have fixed this now :). Please take another look when you can.